app/styles/_data-toolbar.less

```diff
@@ -26,13 +32,20 @@
       .flex(@columns: 1 1 0%);
     }
     @media (min-width: @screen-md-min) {
-      max-width: 600px;
+      max-width: @data-toolbar-filter-max-width;
     }
     .filter .navbar-filter-widget .label-filter .selectize-control .selectize-input {
       &, & input {
         font-size: @font-size-base;
       }
     }
+    .label-filter-key .selectize-input.not-full {
+      // Increase default input click target
+      width: @label-filter-default-input-xs;
+      @media (min-width: @screen-xs) { width: @label-filter-default-input-sm;}
+      @media (min-width: @screen-sm) { width: @label-filter-default-input-md;}
+      @media (min-width: @screen-md) { width: @label-filter-default-input-lg;}
```

Comment:
Shouldn't these media queries use `@screen-xs-min`, `@screen-sm-min` and `@screen-md-min` ?